File: truss-compiler/src/main/java/truss/compiler/symbols/FieldSymbol.java
package truss.compiler.symbols;

public class FieldSymbol extends Symbol {
    @Override
    public SymbolKind getKind() {
        return SymbolKind.FIELD;
    }

    @Override
    public void accept(SymbolVisitor visitor) {
        if (!visitor.isDone()) {
            visitor.visitField(this);
        }
    }
}

File: Truss.Compiler/Symbols/FieldSymbol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Truss.Compiler.Symbols {
    public class FieldSymbol : Symbol {
        public override SymbolKind Kind {
            get { return SymbolKind.Field; }
        }

        public override void Accept(ISymbolVisitor visitor) {
            if (!visitor.Done) {
                visitor.VisitField(this);
            }
        }
    }
}
